mods/cnc/rules/structures.yaml

```diff
@@ -72,6 +72,7 @@ FACT:
 	WithBuildingPlacedAnimation:
 	Power:
 		Amount: 0
+	ProvidesCustomPrerequisite@buildingName:
```

Comment:
The above statement is moot so it can be ignored, though I am going to ask that this becomes `@actor-name` to follow what we do in other places.